console-subscriber/Cargo.toml

```diff
@@ -22,4 +22,12 @@ futures = { version = "0.3", default-features = false }
 tokio = { version = "1", features = ["full", "rt-multi-thread"]}
 futures = "0.3"
 
-tracing-subscriber = { version = "0.2.17", features = ["fmt", "registry", "env-filter"] }
\ No newline at end of file
+tracing-subscriber = { version = "0.2.17", features = ["fmt", "registry", "env-filter"] }
+
+[[example]]
+name = "app"
+path = "examples/app.rs"
+
+[[example]]
+name = "dump"
+path = "examples/dump.rs"
```

Comment:
are these necessary for the examples to work? i know i've used `cargo run --example` to run them locally without having to add this to the Cargo.toml...

Reply:
Same, I also run (from the `console-subscriber` directory) `cargo run --example app` and it works...

Reply:
> Same, I also run (from the `console-subscriber` directory) `cargo run --example app` and it works...

you should be able to do it even from the repo root, at least on Rust 1.52.0 --- i've been doing this

Reply:
yes in deed, this is the case. Will update